dvc/command/metrics.py

```diff
@@ -12,7 +12,46 @@ def _show(self, metrics):
                 logger.info("{}:".format(branch))
 
             for fname, metric in val.items():
-                logger.info("\t{}: {}".format(fname, metric))
+                self._output_formatted_metric(fname, metric)
+
+    def _is_json(self, content):
+        import json
+
+        try:
+            json.loads(content)
+            return True
+        except ValueError:
+            return False
+
+    def _output_formatted_metric(self, fname, metric):
+        import csv
+
+        if "\n" not in metric:
+            logger.info("\t{}: {}".format(fname, metric))
+            return
+
+        logger.info("\t{}:".format(fname))
+
+        if self._is_json(metric):
+            for line in metric.split("\n"):
```

Comment:
why do we need this split? :) I'm not sure I understand

Reply:
Because we want to print the JSON file line by line to create the following output:
```
	metrics.json:
		{
		   "deviation_mse": [
		      "0.173461",
		      "0.289545",
		      "0.297848"
		   ],
		   "data_set": [
		      "train",
		      "testing",
		      "validation"
		   ],
		   "value_mse": [
		      "0.421601",
		      "0.67528",
		      "0.671502"
		   ]
		}
```

If we don't do it, there's no way that I know to append `\t\t` to every line.